packages/visual-editor/src/components/puck/Breadcrumbs.tsx

```diff
@@ -40,7 +40,8 @@ export const BreadcrumbsComponent = (props: BreadcrumbsProps) => {
   const { separator = "/" } = props;
   const { document, relativePrefixToRoot } = useTemplateProps<any>();
   let breadcrumbs = getDirectoryParents(document);
-  if (breadcrumbs) {
+  // if no dm_directoryParents nor dm_directoryChildren, then don't append current
```

Comment:
Specify that the children check is so that the non-linked root breadcrumb shows up on the root DM page but doesn't show on the entity page